variants/esp32c2/pins_arduino.h

```diff
@@ -0,0 +1,29 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+
+static const uint8_t LED_BUILTIN = SOC_GPIO_PIN_COUNT+13;
+#define BUILTIN_LED  LED_BUILTIN // backward compatibility
+#define LED_BUILTIN LED_BUILTIN  // allow testing #ifdef LED_BUILTIN
+
+static const uint8_t TX = 20;
+static const uint8_t RX = 19;
+
+static const uint8_t SDA = 8;
+static const uint8_t SCL = 9;
+
+static const uint8_t SS    = 4;
+static const uint8_t MOSI  = 7;
+static const uint8_t MISO  = 5;
+static const uint8_t SCK   = 6;
```

Comment:
Let SPI pins be the same as C3 has:
```suggestion
static const uint8_t SS    = 7;
static const uint8_t MOSI  = 6;
static const uint8_t MISO  = 5;
static const uint8_t SCK   = 4;
```